fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -11880,22 +11930,22 @@
 | `value=(empty string)`| (initial)| `<empty string>` |
 | `value=(array with string)`| (changed)| `"string"` |
 | `value=(empty array)`| (initial)| `<empty string>` |
-| `value=(object)`| (changed)| `"result of toString()"` |
+| `value=(object)`| (changed, ssr error, ssr mismatch)| `"result of toString()"` |
```

Comment:
This is a new throw. I'll add this to a list of things to check before the release.